README.md

```diff
@@ -1,25 +1,35 @@
-# COMIT-rs
+<a href="https://comit.network">
+<img src="logo.svg" height="120px">
+</a>
+
+---
+
+[COMIT](https://comit.network) is an open protocol facilitating cross-blockchain applications.
+With [COMIT](https://comit.network) you can for examples exchange Bitcoin for any Erc20 token directly with another person.
+
+This repository containers the implementation of the comit-network daemon (`cnd`), which is the reference implementation of the protocol in Rust. 
+
+If you wish to do an atomic swap on your machine or to integrate COMIT into an application (e.g. a DEX) please take a look at the [Getting Started section](https://comit.network/docs/getting-started/create-comit-app/) of the COMIT documentation.
+If you have any question please [reach out to the team in our Gitter chat](https://gitter.im/comit-network/community)!
```

Comment:
This would reduce repetition compared to the previous sentence.

```suggestion
If you have any questions, feel free to [reach out to the team in our Gitter chat](https://gitter.im/comit-network/community)!
```